docs/DeveloperGuide.md

```diff
@@ -51,9 +51,19 @@ For example, the `Logic` component (see the class diagram given below) defines i
 
 **How the architecture components interact with each other**
 
-The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The *Sequence Diagram*s below show how the components interact with each other for the scenario where the user issues the commands 
 
-<img src="images/ArchitectureSequenceDiagram.png" width="574" />
+* `delete 1` (in deliverable or meeting mode)
+
+![`Architecture Sequence Diagram with Dashboard](images/ArchitectureSequenceDiagramWithDb.png)
```

Comment:
is Save to File a real method or these short forms are allowed? not sure

Reply:
Ahh it's unedited from AB3